### ISSUE TITLE
Add writer params and function to widget

### ISSUE THREAD

**Issue by JacksonMaxfield** [AllenCellModeling/napari-aicsimageio#33]
@psobolewskiPhD I thought of yet another feature to add to this widget if no one complains about it for lets say ~1 month?

When the widget is loaded, we can add params and a button to actually run the function to save some set of the loaded layers as channels to an OME-* file (OME-TIFF for right now but OME-Zarr in the future).

**Comment by psobolewskiPhD** [comment]
Yes, re-saving the selected scenes would be a nice GUI feature. 
While I love the fact that with AICSImageIO I can just use the native LIF files, without in-between files that take up space and potentially obscure the data origin, it can be very useful to be able to pull a few scenes to a new file/format to share/whathave you.

Another thing—maybe already possible, via viewer.layers.data, frankly I'm not sure, just thinking aloud—is to let the user return a dask array (or xarray?) stack of the selected scenes. What I do now is browse the scenes via drag-n-drop and then build my dask stack as you showed me previously. Once I have the stack, I run my StarDist or pyCLesperanto analysis code on it.

**Comment by psobolewskiPhD** [comment]
FYI: readlif has been updated to now provide the more verbose scene naming. I updated AICSImageIO, but it's peachy with an older version. Maybe consider bumping the requirements? The more verbose Scene names are pretty nice with the new scene selector....

**Comment by JacksonMaxfield** [comment]
> FYI: readlif has been updated to now provide the more verbose scene naming. I updated AICSImageIO, but it's peachy with an older version. Maybe consider bumping the requirements? The more verbose Scene names are pretty nice with the new scene selector....

Mind making a PR to AICSImageIO? I started PhD this week so am a bit slammed with getting up to speed on twenty different things :sweat_smile:
